Bottom: d9dd29d7183680f1ec1c644c3e47f04c9840abe5
Top:    73afcbe9293edf53861a49c2afbaec5a72b4a6c5
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-13 18:34:46 +0100

Refresh of add-autoconf-tests-for-gl-and-

---

diff --git a/autogen.sh b/autogen.sh
index 3eeeca2..166a3d0 100755
--- a/autogen.sh
+++ b/autogen.sh
@@ -84,7 +84,7 @@ rm -f po/Makefile.in.in.orig
 #
 
 echo "Running aclocal..."
-aclocal $ACLOCAL_FLAGS || exit 1
+aclocal -I m4 $ACLOCAL_FLAGS || exit 1
 echo "Done with aclocal"
 
 echo "Running autoheader..."
